src/vectorchord/advanced-features/rerank-in-table.md

```diff
@@ -0,0 +1,33 @@
+# Rerank In Table <badge type="tip" text="since v0.3.0" />
+
+To maximize performance, VectorChord stores vectors in both the index and the table.
+Although vectors can also be fetched from the table, doing so is much slower than fetching them from our highly optimized data structure.
+
+As a result, VectorChord's [superior performance](https://blog.vectorchord.ai/vector-search-over-postgresql-a-comparative-analysis-of-memory-and-disk-solutions) is partly due to its larger disk usage compared to disk-saving solutions like pgvectorscale. However, this is usually acceptable because disk space is affordable, and performance is more important.
+
+This behavior (vectors are stored in both the index and the table, and fetched from the index in search) is the default option for VectorChord.
```

Comment:
Although it has been marked as resolved, I still remember my previous conversion. Don’t put these two things side by side. It’s a trick, just a trick.